test/test_functional.py

```diff
@@ -0,0 +1,94 @@
+import os
+
+import torch
+import torchaudio
+import unittest
+import test.common_utils
+
+
+class TestFunctional(unittest.TestCase):
+    # size (2,20)
+    test_data = torch.tensor([
+        [45.4243, 81.9316, 19.1100, 32.4998, 45.3313, 68.8204, 42.0782, 19.7222,
+         76.8721, 69.9104, 27.7188, 86.3579, 30.3251, 92.0308, 70.0568, 74.8940,
+         94.3127, 82.9875, 88.8303, 96.3460],
+        [59.4262, 91.0040, 74.7672, 79.8533, 46.7943, 13.6757, 85.5145, 33.0060,
+         88.5102, 25.6912, 57.9501, 33.3326, 71.5654, 90.0321, 81.8218, 91.6907,
+         87.9834, 16.4177, 62.4474, 0.2146]
+    ]).float()
+
+    def _test_istft_helper(self, sound, kwargs):
+        stft = torch.stft(sound, **kwargs)
+        estimate = torchaudio.functional.istft(stft, length=sound.size(1), **kwargs)
+
+        # trim sound for case when constructed signal is shorter than original
+        sound = sound[:, :estimate.size(1)]
+
+        self.assertTrue(sound.shape == estimate.shape, (sound.shape, estimate.shape))
+        self.assertTrue(torch.allclose(sound, estimate, atol=1e-4))
+
+    def test_istft(self):
+        kwargs1 = {
+            'n_fft': 12,
+            'hop_length': 4,
+            'win_length': 12,
+            'window': torch.hann_window(12),
+            'center': True,
+            'pad_mode': 'reflect',
+            'normalized': True,
+            'onesided': True,
+        }
+
+        kwargs2 = {
```

Comment:
Also add some coverage of edge cases. Like empty input, window length shorter than hop length, n_fft bigger than window length, etc. - You can also check if the test predictably fails for cases in which the window isn't appropriate.

Also, add some comments as to why you chose these parameters and not some other ones specifically.